src/core/client/events/keyHeld.ts

```diff
@@ -2,33 +2,33 @@ import alt from 'alt-client';
 
 const holdableKeys: { [key: string]: Array<{ onKeyDown?: Function; onKeyUp?: Function }> } = {};
 
-class InternalFunctions {
-    static handleKeyUp(key: number) {
+const InternalFunctions = {
```

Comment:
Should be i.e. const InternalFunctionsConst = {} 

and then you'd have to export it at the end of the file like ```ts
export const InternalFunctions = {
    ...InternalFunctionsConst
}``` 

I think its not accessible by this Pull Request in its current state.

See https://github.com/Stuyk/altv-athena/issues/299 for more.

Reply:
Internal functions aren't meant to be exposed. @Booster1212